proto-types/build.rs

```diff
@@ -1,8 +1,8 @@
 use std::{env, error::Error};
 fn main() -> Result<(), Box<dyn Error>> {
-    let account_address_prefix = env::var("ACCOUNT_ADDRESS_PREFIX").map_err(|_| "ACCOUNT_ADDRESS_PREFIX environment variable must be set. This is best done in a .cargo/config.toml file in the root of your project")?;
+    let account_address_prefix = env::var("BECH_32_MAIN_PREFIX").map_err(|_| "BECH_32_MAIN_PREFIX environment variable must be set. This is best done in a .cargo/config.toml file in the root of your project")?;
     println!(
```

Comment:
Is final application should have this `println!` too? If not, you probably need to use `dbg!`

Reply:
If those env variables are not set then the error message is displayed when building an application. Is that what you're concerned about?